core/chaincode/lifecycle/lifecycle_test.go

```diff
@@ -385,21 +385,16 @@ var _ = Describe("ExternalFunctions", func() {
 			})
 		})
 
-		When("the chaincode is already being built", func() {
+		When("the chaincode build previously failed", func() {
```

Comment:
I would have thought to leave this case the same, but split it into two scenarios, one where it does not attempt to build because the previous build was successful, and another where the previous build failed and it does.